test/game_rules_test.exs

```diff
@@ -5,25 +5,41 @@ defmodule GameRulesTest do
 
   setup :verify_on_exit!
 
-  test "a game is over if a bomb has been selected" do
+  test "the player has lost if a bomb has been selected" do
```

Comment:
I know we suggested removing the duplication caused by having to explicitly define what the `MockRandomizer` should return, but now I think this test is a bit hard to read. What do you think? Is there a happy medium?